### PR TITLE
Create installation section

### PR DESCRIPTION
**Pull request by geriom**
# Changes

To have a better docs user experience, this new section surfaces the installation documentation for all the Tekton components.

This first iteration adds Pipelines and Triggers, more work is required for other components' installation docs to be included here.

Screenshot from 2022-09-27 11-55-26

# Submitter Checklist

These are the criteria that every PR should meet, please check them off as you
review them:

- [x] Includes [docs](https://github.com/tektoncd/community/blob/master/standards.md#principles) (if user facing)
- [x] Commit messages follow [commit message best practices](https://github.com/tektoncd/community/blob/master/standards.md#commit-messages)

_See [the contribution guide](https://github.com/tektoncd/website/blob/master/CONTRIBUTING.md)
for more details._

### CODE REVIEW

**Review by dibyom — APPROVED** [review]

**Comment by geriom** [comment]
> > more work is required for other components' installation docs to be included here.
> 
> Can you elaborate on this?

@AlanGreene  not all repos have an `install.md` file, chains and CLI for instance have their installation instructions in their README. I started with Pipelines and Triggers because they are the most critical. I would also like to implement some changes so all the installation docs have the same structure and style.

File: content/en/docs/Installation/local-installation.md [hunk 1]
```diff
@@ -0,0 +1,99 @@
...
+title: "Local kubernetes cluster"
+linkTitle: "Local cluster"
+weight: 1
+description: >
+  Set up a Kubernetes cluster on your computer to test Tekton.
+---
+-->
+
+If you are going to test Tekton locally on your computer, two possible
+alternatives for that are **minikube** and **kind**.
```
Comment (afrittoli) [inline content/en/docs/Installation/local-installation.md:12]: There are other alternatives as well, like k3c, but of course we do not need to document all.
The problem I see here is that these are vanilla instructions, there's no Tekton specific setup / tuning, so I don't see the value in maintaining them here as opposed to just link to `kind` and `minikube` docs.

If we were to say how setup a cluster for Tekton development, or how to setup a cluster with a local registry for pushing images in Tekton tasks, I think there'd be more value, but as it is I don't see this content providing help.

File: content/en/docs/Installation/pipelines.md [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+---
+title: "Install Tekton Pipelines"
+linkTitle: "Tekton Pipelines"
+weight: 2
+description: >
+  Install Tekton Pipelines on your cluster
+---
+-->
+
+{{% readfile "/docs/Pipelines/install.md" %}}
```
Comment (afrittoli) [inline content/en/docs/Installation/pipelines.md:11]: Ah, nice, I didn't know you could do this.
I think we should mention somewhere that this is the install guide for Tekton on the `main` branch.
Version-specific guides are available on GitHub.

**Review by afrittoli — COMMENTED** [review]
Thanks for this. 

I think this it's really useful to have these collected in one section, and we can probably work with the different projects to get some alignment in terms of file names and locations.

I think the local setup section is not ideal as it is though, I would probably only include links to the tools (minikube, kind) and to the install guides.
It may be useful to include some sizing suggestions instead, like how much memory / disk / CPU.

This is what I use to run a 3 nodes cluster that runs Tekton and a bunch of other things so it can probably be smaller than this

```json
          "Resources": {
               "CPUs": 4,
               "DiskSize": 100,
               "Memory": 8192
          },
```

File: content/en/docs/Installation/pipelines.md [hunk 3]
```diff
@@ -0,0 +1,17 @@
+<!--
+---
+title: "Install Tekton Pipelines"
```
[the remark this answers is not in the document]
Comment (geriom) [reply content/en/docs/Installation/pipelines.md:3]: Yes, that's the reason, my plan is to submit some updates later on to that repo. I already have a draft in progress.

File: content/en/docs/Installation/pipelines.md (hunk 2 above)
Comment (afrittoli) [reply content/en/docs/Installation/pipelines.md:11]: Ah, cool, I didn't realise this, nice work!

**Comment by geriom** [comment]
> I think the local setup section is not ideal as it is though, I would probably only include links to the tools (minikube, kind) and to the install guides. It may be useful to include some sizing suggestions instead, like how much memory / disk / CPU.

@afrittoli  I agree, there's no value in repeating the installation instructions for minikube and kind. I simplified the content to link to the corresponding docs, and made the links to the setup and scripts more prominent. TAL

**Review by afrittoli — COMMENTED** [review]
Thank you @geriom for the updates, it looks good to me!

Once thing that I noticed is that when one switches version, it goes to the homepage for that version and there is no side menu there.  Not something for this PR, but something that would be good to fix. 

I'm not 100% sure but I feel like the left menu used to be there, perhaps that's something that changed with the docsy theme update?

/approve
/lgtm

**Pull request merged**